standard/standard_1.php

```diff
@@ -1069,7 +1069,7 @@ function implode (array|string $separator = "", ?array $array): string
  * elements in the same order, with the glue string between each element.
  */
 #[Pure]
-function join (array|string  $separator = "", ?array $array): string
+function join (array|string  $separator = "", ?array $array = null): string
```

Comment:
This is not correct: https://3v4l.org/n3SIL

Reply:
Reverted change and added to ignore